openedx/core/djangoapps/user_authn/config/waffle.py

```diff
@@ -32,3 +32,14 @@
 ENABLE_PWNED_PASSWORD_API = WaffleSwitch(
     f'{_WAFFLE_NAMESPACE}.enable_pwned_password_api', __name__
 )
+
+# .. toggle_name: user_authn.enable_country_disabling
+# .. toggle_implementation: WaffleSwitch
+# .. toggle_default: False
+# .. toggle_description: Waffle flag to enable the feature of registration login from specific countries
+# .. toggle_use_cases: temporary, open_edx
+# .. toggle_creation_date: 2021-09-22
+# .. toggle_target_removal_date: 2021-12-31
+# .. toggle_warning: When the flag is ON, the feature of disabling registration from specific countries is enabled.
+# .. toggle_tickets: VAN-664
```

Comment:
Why is this toggle necessary? Seems it would be simpler to just let `DISABLED_COUNTRIES == []` mean that no countries are disabled.

Reply:
Hi, I have removed the waffle flag, thanks for reviewing.

Reply:
Thanks @AhtishamShahid